wikibase-datamodel-tests.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import {
+	DataType,
+	DataValueType,
+	SnakType,
+	Statement,
+	StatementMap,
+} from '.';
+
+const douglasAdamsStatement: Statement = {
+	mainsnak: {
+		snaktype: SnakType.value,
+		property: 'P373',
+		datatype: DataType.string,
+		datavalue: {
+			type: DataValueType.string,
+			value: 'Douglas Adams',
+		},
+	},
+	type: 'statement',
+	rank: 'normal',
```

Comment:
Maybe I'm missing a puzzle piece but my attempts of getting CI for this (#4) wanted this to look like the other uses, e.g. L16, rather than a string literal.

```suggestion
	rank: Rank.normal,
```
(warning: needs an extra import)

Reply:
Yeah, I only migrated `Rank` to an enum late in the evolution of this change, and I guess I never ran the tests after that. Thanks!